### DISCUSSION TITLE
Redis Password

### DISCUSSION THREAD

**Discussion by Collectathon** [MDeLuise/plant-it#246] [Q&A] [answered]
Hello! I am trying to create an Unraid template for this project and would like to know if it is possible to pass through a password for Redis.

**Comment by MDeLuise** [comment, marked as answer]
Hi @Collectathon, thanks for asking this.
This feature has been completed and will be implemented in the next project release.
I added the property `CACHE_PASSWORD` in order to specify that.

**Reply by jonzey231** [reply]
Was an Unraid template ever created? I've tried installing via docker compose and get error after error. A template in Community Apps would be wonderful.

**Reply by Collectathon** [reply]
Thanks for reminding me this exists. I never actually ended up doing it.
I did spend time today putting a template together, and it should be available shortly in the Unraid CA.

**Reply by jonzey231** [reply]
Hey that's awesome. Thanks!

**Reply by jonzey231** [reply]
The db fields show MySQL but can we also use MariaDB and just put the info in those same fields?

**Reply by Collectathon** [reply]
Yes, that's what I have been using.

**Reply by jonzey231** [reply]
Question. The port selections are a bit different than most CA templates. 

The WebUI loads on :3000 but the server is at :8090 (in my case). I got it all setup with a user and now I want to reverse proxy it so I can save/install it as a PWA on mobile. 

So I setup my reverse proxy to point at 192.168.50.163:8090 and I get the "Hi friend!" page. Then I enter my local IP:Port (http://192.168.50.163:8090/) but it says it cannot find the server.

I'm just not sure where to go from here to make it accessible via reverse proxy.

**Reply by jonzey231** [reply]
Update. 

Got it. Just had to reverse proxy the server (:8090) as well and use _that_ URL.
